packages/core/solid/use-chat.ts

```diff
@@ -76,6 +76,7 @@ export function useChat({
   initialInput = '',
   sendExtraMessageFields,
   experimental_onFunctionCall,
+  experimental_onToolCall,
```

Comment:
This will be deprecated very soon (as will onFunctionCall). It's an old implementation that's not compatible with AI SDK 3.1 - I'm working on a replacement.

I'd prefer if we wouldn't add this here.